### DISCUSSION TITLE
I somewhy can't publish my website

### DISCUSSION THREAD

**Discussion by ghost** [community/community#38304] [Other Feature Feedback, Questions, & Ideas]
I'm trying to publish a website for school project, but when choosing website link it keeps giving out the same error (Both radioguitars.com and its alternate name are improperly configured
Domain's DNS record could not be retrieved. For more information, see [documentation](https://docs.github.com/articles/setting-up-a-custom-domain-with-github-pages/) (InvalidDNSError).) I've tried cleaning my browser's cache

**Comment by airtower-luna** [comment]
I don't see any DNS record for that domain. Did you actually buy it? You need to do that before you can set it up for Pages. :thinking:
